content/300-guides/200-deployment/201-serverless/500-deploy-to-netlify.mdx

```diff
@@ -18,14 +18,14 @@ Before you can follow this guide, you will need to set up your application to be
 
 ## Binary targets in <inlinecode>schema.prisma</inlinecode>
 
-The Prisma schema should contain the following in the `generator` block:
+Since your code is being bundled and deployed to Netlify's environment, which isn't necessarily the same as your local environment, you will need to set [`binaryTargets`](/reference/api-reference/prisma-schema-reference#binarytargets-options) in order to bundle the query engine that is compatible with Netlify. If you do not set this option, your deployed code will have an incorrect query engine deployed with it and will not function.
```

Comment:
```suggestion
Since your code is being deployed to Netlify's environment, which isn't necessarily the same as your local environment, you will need to set [`binaryTargets`](/reference/api-reference/prisma-schema-reference#binarytargets-options) for Netlify build step in order to download the query engine that is compatible with Netlify runtime. If you do not set this option, your deployed code will have an incorrect query engine deployed with it and will not function.
```

Reply:
I think we can avoid saying `bundle` here, the engine is not bundled actually, but installed in their build runner.